r/R/arrow-package.R

```diff
@@ -103,6 +104,10 @@ arrow_info <- function() {
         bytes_allocated = pool$bytes_allocated,
         max_memory = pool$max_memory,
         available_backends = supported_memory_backends()
+      ),
+      runtime_info = list(
+        simd_level = runtime_info()[1],
```

Comment:
Don't call this function twice (though it should be cheap enough); treat it like how we treat `default_memory_pool()` above.